helm-charts/support/values.yaml

```diff
@@ -100,6 +100,26 @@ prometheus:
       - key: k8s.dask.org/dedicated
         value: worker
         effect: NoSchedule
+    # resources for the node-exporter was set after inspecting cpu and memory
+    # use in a few clusters using prometheus queries from
+    # https://gist.github.com/payam-int/edf977c6af603fee0ce1b05da7792fe7.
```

Comment:
remove the trailing . so the link can be copy pasted easily

Reply:
I realized that what I've copied from there was cpu/memory usage of node exported by node exporter, rather than the usage of the node-exporter pod itself. Due to this, I've reworked this PR and no longer link out.